packages/teams-js/src/public/app.ts

```diff
@@ -488,12 +488,12 @@ export namespace app {
     sharepoint?: any;
 
     /**
-     * Info of the sharePoint site associated with the team.
+     * When running in Teams for an organization with a tenant, provides information about the SharePoint site associated with the team.
```

Comment:
Interesting that some of these say "else `undefined`" and some don't, is it intentional?

Reply:
Not intentional, and now fixed.